docs/operators.md

```diff
@@ -33,26 +33,15 @@ Additionally, they do not have an intrinsic cost, since they are compiled to CLV
 | +        | `(+ A B ...)`  | Adds multiple atoms.                            |
 | -        | `(- A B ...)`  | Subtracts multiple atoms.                       |
 | \*       | `(* A B ...)`  | Multiplies multiple atoms.                      |
-| /        | `(/ A B)`      | Divides two atoms.                              |
+| /        | `(/ A B)`      | Divides two atoms. Rounds towards negative      |
+|          |                | infinity.                                       |
```

Comment:
This seems to be creating a new row (according to https://github.com/Chia-Network/chialisp-web/blob/647902983eee1874a89890e798278a7cfc7cc3c6/docs/operators.md).